packages/expo-updates/src/__tests__/Updates-test.ios.ts

```diff
@@ -58,3 +59,39 @@ it('returns the proper object in fetchUpdateAsync if an update is available and
   const expected = { isNew: true, manifest: fakeManifest };
   expect(actual).toEqual(expected);
 });
+
+it('returns the proper object when no logs from readLogEntriesAsync', async () => {
```

Comment:
we should add a similar test in Updates-test.android.ts

Reply:
Sadly, Updates-test.android.ts does not exist...